### ISSUE TITLE
Place demos into an installer, and remove Nuget references to aid developer use

### ISSUE THREAD

**Issue by Smurf-IV** [Wagnerp/Krypton-NET-5.470#133]
When I want to check something with the latest build, I have to keep removing the nuget package from the demo app's
It would be better to have a separate installer / nuget with just the demo exe's and their required Krypton dll's.
And then the demo projects can use the latest built on a developer machine.

**Comment by Wagnerp** [comment]
I think updating the seperate installer would be better for the demo apps, as if they were to be packaged with the NuGet package, it would bloat the file size to ~150 MB.

**Comment by Wagnerp** [comment]
Although, saying that, I could create a NuGet package `KryptonToolkitSuite54xxDemos` with just the demos in it?

**Comment by Wagnerp** [comment]
Strip out the DLLs?

**Comment by Smurf-IV** [comment]
> 
> 
> Strip out the DLLs?

No, That would just cause more confusion for a starter.
Separate nuget / msi with just compressed bin's and dll's from the one directory.
Then make sure the launch "browser" app works from that same directory.

**Comment by Wagnerp** [comment]
Would you mind testing out the new installer, to see if I missed anything out?

**Comment by Smurf-IV** [comment]
> 
> 
> Would you mind testing out the new installer, to see if I missed anything out?

Is this the release (Needs another name if it is;) or do I have to get the code and build it ?

**Comment by Wagnerp** [comment]
No, I built the binaries, so no need to build from code.

**Comment by Smurf-IV** [comment]
```Error
Install-Package : Could not install package 'KryptonToolkitSuite5470Demos 17.470.6015.15'. You are 
trying to install this package into a project that targets '.NETFramework,Version=v4.7.2', but the 
package does not contain any assembly references or content files that are compatible with that 
framework. For more information, contact the package author.
At line:1 char:1
+ Install-Package KryptonToolkitSuite5470Demos -Version 17.470.6015.15
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Not sure I "Like" a nuget for installing demo's into an existing solution.. These should be a standalone installer, via the releases, with a link from the ReadMe page.

**Comment by Wagnerp** [comment]
Yes I agree, will put a link to the releases tab on the readme and will de-list KryptonToolkitSuite5470Demos.

**Comment by Smurf-IV** [comment]
Completed #130 Projects ending in 2018 can be deleted ? 
- Remove Nuget references for Krypton.
- Add direct references to the Krypton Component bin output directory

**Comment by Wagnerp** [comment]
> Remove Nuget references for Krypton.

Do you mean from the demos?

**Comment by Smurf-IV** [comment]
Yes, Only Krypton references, So when they are built they will be using whatever is on the developers machine.

**Comment by Smurf-IV** [comment]
![image](https://user-images.githubusercontent.com/2418812/56156805-e491f200-5fb5-11e9-88b4-0583771337d8.png)

**Comment by Wagnerp** [comment]
Works for me, did you get a UAC prompt when you clicked `install`?

**Comment by Wagnerp** [comment]
Hi @Smurf-IV 

Now I figured out how to get the version of the DLL, should I make a new `About` tab which includes the version of all 6 DLLs?

![image](https://user-images.githubusercontent.com/949607/56293301-b977e180-6120-11e9-9b32-cce1cbbff884.png)

**Comment by Wagnerp** [comment]
Hi @Smurf-IV 

I may clone the demo code into a new `NuGet Enabled` directory, so I can easily build/release new installers, this will preserve the non-NuGet code.
